src/transformers/models/dinov2/modeling_dinov2.py

```diff
@@ -858,8 +892,9 @@ def forward(
                 output = (feature_maps,) + outputs[2:]
             return output
 
-        return BackboneOutput(
+        return Dinov2BackboneOutput(
```

Comment:
this is kind of breaking if the tuples have the same behaviour, and not very consistent in case `cls_tokens` can only be return if `return_dict = True`. But better than breaking 😉